pkg/controllers/clusterresourceplacement/controller.go

```diff
@@ -216,9 +219,24 @@ func (r *Reconciler) handleUpdate(ctx context.Context, crp *fleetv1beta1.Cluster
 		return ctrl.Result{RequeueAfter: 5 * time.Minute}, nil
 	}
 
-	klog.V(2).InfoS("Placement rollout has not finished yet and requeue the request", "clusterResourcePlacement", crpKObj, "status", crp.Status)
+	if !isClusterScheduled {
```

Comment:
Just a clarification: this shortcut would only work when there is no cluster selected at all, right?

Reply:
yeah, now that this only applies when there is no cluster selected then probably not that urgent.